mmv1/products/vertexai/terraform.yaml

```diff
@@ -97,6 +97,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
       name: !ruby/object:Overrides::Terraform::PropertyOverride
         custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+      monitoringConfig.snapshotAnalysis.stalenessDays: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
[MEMO] https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.featurestores.entityTypes#snapshotanalysis says like below. Therefore, I decided to let the terraform calculate the default value from the API.

> Customized export features time window for snapshot analysis. Unit is one day. Default value is 3 weeks. Minimum value is 1 day. Maximum value is 4000 days.

Reply:
I understand `default_from_api` is for when the default value is not fixed like timestamp. I'll fix the code to use `default_value` instead.

Reply:
I fixed it in 7141575